test/functional/rpc_psbt.py

```diff
@@ -977,6 +981,74 @@ def test_psbt_input_keys(psbt_input, keys):
             assert hash.hex() in res_input[preimage_key]
             assert_equal(res_input[preimage_key][hash.hex()], preimage.hex())
 
+        self.log.info("Test decoding PSBT with MuSig2 per-input and per-output types")
```

Comment:
Nit: Maybe consider putting this in a separate subtest as the current `run_test` is quite long already.

Reply:
Good idea, done.